pkgs/applications/virtualization/singularity/generic.nix

```diff
@@ -44,6 +44,7 @@ in
   gpgme,
   libseccomp,
   libuuid,
+  mount,
```

Comment:
Hmm, I just checked my local system and apparently I've been using `mount` from a security wrapper rather than from the store:

```
❯ which mount | xargs readlink -f
/run/wrappers/wrappers.VdfFiNgLp3/mount
```

Reply:
According to [a StackExchange answer](https://unix.stackexchange.com/questions/107500/why-does-mount-have-the-suid-bit-set/107504#107504), the SUID bit set on the `mount` binary allows unprivileged users to mount/unmount some `/etc/fstab` entries with `user`/`group` options specified. However, I think that's not how it is used in Apptainer and SingularityCE.